UPDATE.md

```diff
@@ -0,0 +1,56 @@
+# Update cloud-provider-gcp with Kubernetes release
+
+## Overview
+
+The GCP cloud provider provides the interface between a Kubernetes cluster and GCP service APIs. 
+This project allows a Kubernetes cluster to provision, monitor and add/remove GCP resources necessary for operation of the cluster. 
+It has to be updated together with Kubernetes releases. 
+This document describes the steps to update cloud-provider-gcp with Kubernetes release.
+
+## Workflow
+
+1. Update library to the latest version.
+`https://github.com/kubernetes/cloud-provider-gcp/blob/master/go.mod` describes the required libraries. 
+   Update the version of each dependency to the latest Kubernetes release version.
+
+
+2. Update library for //providers package.
+   `https://github.com/kubernetes/cloud-provider-gcp/blob/master/providers/go.mod` describes the required libraries for `providers` package.
+   Update the version of each dependency to the latest Kubernetes release version.
+
+
+3. In `https://github.com/kubernetes/cloud-provider-gcp/blob/master/WORKSPACE`, update kube-release sha and version to the latest release version. 
```

Comment:
Where can I find this version? It would be good to add instructions on how to get these values.

Reply:
Added link to kubernetes release page

Reply:
+1, though again I wonder if it makes sense to use desired.

Reply:
Updated